synapse/handlers/pagination.py

```diff
@@ -102,7 +106,7 @@ def __init__(self, hs):
                     job["longest_max_lifetime"],
                 )
 
-    async def purge_history_for_rooms_in_range(self, min_ms, max_ms):
+    async def purge_history_for_rooms_in_range(self, min_ms: int, max_ms: int):
```

Comment:
Can you remove the types from the docstring? (Also these are optional it seems.)

Reply:
Woops